tests/Extensions/Hosting.Services.Web.UnitTests/Middlewares/UserIdentity/UserIdentiyMiddlewareTests.cs

```diff
@@ -10,6 +10,7 @@
 using Microsoft.Omex.Extensions.Hosting.Services.Web.Middlewares;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
+using System.Collections.Generic;
```

Comment:
nit: move `System` using statements to the top

Reply:
Damn you visual studio 😱